ultrasonic/src/main/kotlin/org/moire/ultrasonic/service/MediaPlayerService.kt

```diff
@@ -49,8 +57,7 @@ import timber.log.Timber
  * while the rest of the Ultrasonic App is in the background.
  */
 @Suppress("LargeClass")
-class MediaPlayerService : Service() {
-    private val binder: IBinder = SimpleServiceBinder(this)
+class MediaPlayerService : MediaBrowserServiceCompat() {
```

Comment:
Making this class inherit from `MediaBrowserServiceCompat` is what's suggested in [this doc here](https://developer.android.com/guide/topics/media-apps/audio-app/building-a-mediabrowserservice).

The binder is no longer needed as the `MEdiaBrowserServiceCompat::onBind()` callback is now correct.

Reply:
Actually I think it would be better to create a separate MediaBrowserService in Ultrasonic, which would then call the MediaPlayerService for playback related things, but would contain the browsing related logic.
(Also, I suggest to take a look on the DSub code just for hints/ideas, that is another subsonic fork so not too different from Ultrasonic)